nodewatcher/nodewatcher.py

```diff
@@ -69,8 +66,22 @@ def getConfig(instance_id):
 
         os.rename(tup[1], 'nodewatcher.cfg')
 
-    log.debug("region=%s asg=%s scheduler=%s prox_config=%s idle_time=%s" % (_region, _asg, _scheduler, proxy_config, _scaledown_idletime))
-    return _region, _asg, _scheduler, proxy_config, _scaledown_idletime
+    log.debug("region=%s asg=%s scheduler=%s prox_config=%s idle_time=%s" % (_region, _asg, _scheduler, _proxy_config, _scaledown_idletime))
```

Comment:
I see that jobwatcher and  nodewatcher have a lot of common functions:
`get_asg_name`, `getInstanceId`, `getHostname`, `load_scheduler_module`...
What about moving them in a common utils file?

Reply:
Moved into `utils/utils.py`